src/config.js

```diff
@@ -0,0 +1,8 @@
+export const INDEXER = 'https://like.wancat.cc';
+export const API_PUBLIC = 'https://api.like.co';
+
+// The like.co API address
+export const IGNORE_LIST = [
+  'like17m4vwrnhjmd20uu7tst7nv0kap6ee7js69jfrs', 
+  'like1yney2cqn5qdrlc50yr5l53898ufdhxafqz9gxp',
+];
```

Comment:
I would prefer the constant names to include the type / category of the value, e.g. `INDEXER_URL`, `API_PUBLIC_URL`, `IGNORE_ADDRESS_LIST`, so it is easier to know the meaning (e.g. is `INDEXER` the url or the axios instance?) without needing to change the name when importing into other modules.

(This is less important in statically typed languages, since the type itself already gives some info / context, but in dynamically typed languages like JS, the context is exactly what we need in mind when coding)